app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  before_save{email.downcase!}
+  validates :name, presence: true, length: {maximum: 50}
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
+  validates :email, presence: true, length: {maximum: 255},
+                       format: {with: VALID_EMAIL_REGEX},
+                       uniqueness: {case_sensitive: false}
```

Comment:
uniqueness này để làm gì em

Reply:
Dạ theo em thì uniqueness dùng để xác thực email là duy nhất trong trường hợp được in hoa, tham số case_sensitive :false để hàm uniqueness kiểm tra nếu như email chữ in hoa với email chữ thường thì nó vẫn là 1